doc/source/developing.rst

```diff
@@ -95,12 +95,12 @@ If you worked on the documentation, we suggest you verify the markup
 and the result by building the documentation locally and view your
 results.
 
-For building the documentation for the moment you still need buildout (untested)::
+For building the documentation::
 
-    $ python3.8 -mvenv .
-    $ bin/pip install zc.buildout setuptools
-    $ bin/buildout install sphinx
-    $ bin/sphinx
+    $ python3.9 -mvenv .
```

Comment:
`-m venv` with a space? At least, that's what I normally do.

Reply:
Done